examples/Python/PET/listmode_to_sinograms.py

```diff
@@ -77,6 +77,15 @@ def main():
 
     # create listmode-to-sinograms converter object
     lm2sino = pet.ListmodeToSinograms()
+    lm_data = pet.ListmodeData(list_file)
+    sino = pet.AcquisitionData(acq_data_template)
+    rand = pet.AcquisitionData(acq_data_template)
+    sino, rand = lm2sino.sinograms_and_randoms_from_listmode(lm_data, 0, 10, acq_data_template)
+    print(sino.shape)
+    print(sino.norm())
```

Comment:
let's rename `sino` to `prompts`. I'd prefer to rename `rand` to `randoms_estimate` (or at least, say that in a comment). They are not the randoms of course.

Also rename the function to `get_prompts_and_randoms(...)`. However, I'm not a great fan of this I'm afraid. We have `set_input` etc, and here a (presumably static) function that takes an lm_data and time-frame info. I don't see a good reason to have both interfaces. So, I suppose I suggest to have no arguments for this function (i.e. bunch of sets, a set_up(), and then call this). It raises the question why we have it, aside from having a one-liner.

We're mixing 2 kinds of interfaces here, which creates confusion.

If you don't like the `set` functions. we could have a constructor function for Listmode2Sinograms
```python
def __init__(self, list_file, interval, template)
```

Same comments will apply on the C++ side of course.